src/OrchardCore/OrchardCore.DisplayManagement/Descriptors/ShapeTemplateStrategy/BasicShapeTemplateHarvester.cs

```diff
@@ -31,17 +31,16 @@ public IEnumerable<HarvestShapeHit> HarvestShape(HarvestShapeInfo info)
             }
         }
 
-        static string Adjust(string subPath, string fileName, string displayType)
-        {
-            var leader = "";
-            if (subPath.StartsWith("Views/") && subPath != "Views/Items")
-            {
-                leader = subPath.Substring("Views/".Length) + "_";
-            }
+        public virtual bool InDepth => false;
 
+        protected virtual string leader(string subPath) => 
+            (subPath.StartsWith("Views/") && subPath != "Views/Items") ? subPath.Substring("Views/".Length) + "_" : "";
+
+        string Adjust(string subPath, string fileName, string displayType)
```

Comment:
`private string Adjust()`

Currently, in the dev branch, it is `private static string Adjust()` but here can't be static